http_check/datadog_checks/http_check/http_check.py

```diff
@@ -303,7 +303,10 @@ def check_cert_expiration(self, instance, timeout, instance_ca_certs):
 
             ssl_sock = context.wrap_socket(sock, server_hostname=server_name)
             cert = ssl_sock.getpeercert()
-            exp_date = datetime.strptime(cert['notAfter'], "%b %d %H:%M:%S %Y %Z")
```

Comment:
> This prevents the KeyError that would otherwise return when attempting to retrieve the expiration date from a non-existent cert.

Wouldn't the key error be captured on the try/except block this code is already in?